spec/jobs/facilities/facility_location_download_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Facilities::FacilityLocationDownload, type: :job do
+  describe 'NCA Facilities' do
+    it 'retrieves and persists facilities data' do
+      VCR.use_cassette('facilities/va/nca_facilities') do
+        expect(Facilities::NCAFacility.count).to eq(0)
+        Facilities::FacilityLocationDownload.new.perform('nca')
```

Comment:
Can we add "job" or "worker" as a suffix on these class names? I think it will make it more clear that these are workers.